css/structure/jquery.mobile.fixedToolbar.css

```diff
@@ -47,27 +47,27 @@
 .ui-page-footer-fullscreen > .ui-content {
 	padding: 0;
 }
-.ui-fixed-hidden {
+.ui-toolbar-fixed-hidden {
 	position: absolute;
 }
 /* Tap toggle: hide external fixed footer. See issue #6604 */
-.ui-footer-fixed.ui-fixed-hidden {
+.ui-toolbar-footer-fixed.ui-toolbar-fixed-hidden {
   display: none;
 }
-.ui-page .ui-footer-fixed.ui-fixed-hidden {
+.ui-page .ui-toolbar-footer-fixed.ui-toolbar-fixed-hidden {
   display: block
 }
 
-.ui-page-header-fullscreen .ui-fixed-hidden,
-.ui-page-footer-fullscreen .ui-fixed-hidden {
+.ui-page-header-fullscreen .ui-toolbar-fixed-hidden,
+.ui-page-footer-fullscreen .ui-toolbar-fixed-hidden {
```

Comment:
@arschmitz  i guess i missed this... would it be better to switch it to `.ui-toolbar-page...` ? or just drop the `-page` altogether and go with `.ui-toolbar-footer-fullscreen`